scripts/make_tutorials.py

```diff
@@ -61,7 +61,10 @@ class TutorialPage extends React.Component {{
 
 
 def gen_tutorials(
-    repo_dir: str, exec_tutorials: bool, kernel_name: Optional[str] = None
+    repo_dir: str,
+    exec_tutorials: bool,
+    kernel_name: Optional[str] = None,
+    name: Optional[str] = None,
```

Comment:
Could potentially also have `names: Optional[List[str]]` to select more than one tutorial if that would be useful.